package.json

```diff
@@ -55,7 +55,8 @@
   "devDependencies": {
     "@types/node": "^22.14.0",
     "@types/prompts": "^2.4.9",
-    "@typescript-eslint/parser": "^8.26.1",
+    "@typescript-eslint/eslint-plugin": "^8.32.0",
+    "@typescript-eslint/parser": "^8.32.0",
```

Comment:
The latest version has consolidated these in to a single package: https://typescript-eslint.io/getting-started/#step-1-installation